### PR TITLE
2018-12-17 add IFWallet logo to wallets list

### PR DESCRIPTION
**Pull request by bibodeng**
@maplesyrupsucker, IFWallet support BCH, please help me add IFWallet logo to wallet list.

support languages: 简体中文/繁體中文/English

### CODE REVIEW

File: _includes/home/wallets.html [hunk 1]
```diff
@@ -85,9 +85,12 @@ <h2>{% t 'headings.wallets' %}</h2>
             <div class="col-sm-6 col-md-3 vertical-align to-animate">
                 <a href="https://walletgenerator.net/?currency=BitcoinCash" target="_blank"><img alt="WalletGenerator.net" src="/img/wallets/walletgenerator.png"></a>
             </div>
+            <div class="col-sm-6 col-md-3 vertical-align to-animate">
+                <a href="https://www.ifwallet.com.net" target="_blank"><img alt="ifwallet.com" src="/img/wallets/ifwallet.png"></a>
```
Comment (BUYMERCIER) [inline _includes/home/wallets.html]: Incorrect TLD/typo (.com.net)

**Comment by BUYMERCIER** [comment]
@bibodeng check your Telegram messages or hit me up on WeChat using the same handle

File: _includes/home/wallets.html (hunk 1 above)
Comment (bibodeng) [reply _includes/home/wallets.html]: OK, it's spell mistake

**Comment by BUYMERCIER** [comment]
Ah, the structure of the website changed. I am closing this and opening a PR with the appropriate changes myself

**Comment by bibodeng** [comment]
thanks a lot

**Pull request closed without merging**